### DISCUSSION TITLE
Node.js v20.5.1 -> Error: Cannot find module 'socketcluster'

### DISCUSSION THREAD

**Discussion by elegance666** [LiveHelperChat/livehelperchat#1999] [General]
I am trying to enable Node.js server for LiveHelperChat but I have this error: 

`Error: Cannot find module 'socketcluster/fsutil'`

after remove fsutil in server.js I got this error:

`Error: Cannot find module 'socketcluster'`

Which version of node.js will works?

I tried to install that on [NetShekan.com](url)

**Comment by PeopleInside** [comment]
Hi, have you looked at the documentation: https://doc.livehelperchat.com/docs/node-js/ ?
I'm not a developer and I'm not expert on Node.js but I hope the documentation can help you.
